intf/cnsr_intf.lua

```diff
@@ -145,8 +146,21 @@ it runs all the time in the background of VLC, finds the relevant tag and execut
 function looper()
 	local next_loop_time = vlc.misc.mdate()
 	local loop_counter = 0
+	local current_uri = nil
+	
 	while true do
 		if vlc.volume.get() == -256 then break end  -- inspired by syncplay.lua; kills vlc.exe process in Task Manager
+		
+		current_item = vlc.playlist.get(vlc.playlist.current()) -- if a new video is playing, then we want to get the new config
```

Comment:
Should we maybe check once every get loop interval?
meaning move lines 154-162 all of this code into the if loop_counter  == 0